README.md

```diff
@@ -176,7 +176,7 @@ it can't connect to the server a `Redis::CannotConnectError` error will be raise
 ```ruby
 begin
   redis.ping
-rescue Exception => e
+rescue => e
```

Comment:
This is invalid. You should probably use `StandardError`:

```ruby
rescue StandardError => e
```

Reply:
yes, wrt linters, makes sense 👍